src/main/java/duke/core/TypoCorrector.java

```diff
@@ -19,14 +16,10 @@
     private static final double MAX_DISTANCE_DIFF_RATIO = 0.5;
 
     //Sets of "Dictionaries" for the command keyword, categorised by number of keywords contain in a supported commands.
-    private static final ArrayList<String> oneKeywordCommand = new ArrayList<String>(
-            Arrays.asList("bye"));
-    private static final ArrayList<String> twoKeywordsCommands = new ArrayList<String>(
-            Arrays.asList("list patients", "list tasks"));
-    private static final ArrayList<String> otherCommands = new ArrayList<String>(
-            Arrays.asList("update patient", "update task",
-                    "delete patient", "delete task", "add task", "add patient",
-                    "assign by", "find patient", "find task"));
+    private static final String[] simpleCommands = {"bye", "duke", "help", "list patients", "list tasks"};
+    private static final String[] otherCommands = {"update patient", "update task", "delete patient",
+        "delete task", "delete assigned task", "add task", "add patient", "find patient", "find task",
+        "find assigned task", "assign deadline task", "assign event task"};
```

Comment:
`find assigned task` should be `find assigned tasks`, but this is a small fix!

Reply:
Noted! Will fix it in the next PR